.travis.yml

```diff
@@ -9,12 +9,17 @@ rvm:
 jdk:
 - openjdk12
 
+language: node_js
```

Comment:
You can't have multiple language declarations. It's now ignoring the Ruby one and throwing a validation lint on the config. EX: https://travis-ci.com/github/canada-ca/ore-ero/jobs/309080411/config